### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by maxbeatty**
- [x] convert characters.html to JSON ([gist](https://gist.github.com/maxbeatty/db03efcce1bf5daae606deb3c6033703))
- [x] simplify characters script to use JSON
- [x] main.js should spawn and then load "scripts"
- [x] add tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,47 +1,49 @@
...
+    "moduleFileExtensions": [
+      "js",
+      "json"
+    ],
+    "collectCoverageFrom": [
+      "lib/**/*.js"
+    ],
+    "coverageThreshold": {
+      "global": {
+        "branches": 100,
```
Comment (maxbeatty) [inline package.json]: 💯

File: main.js [hunk 2]
```diff
@@ -1,14 +1,9 @@
-const winston = require('winston');
```
Comment (maxbeatty) [inline main.js]: I went back and forth on `winston`. For our needs now and what it was doing (logging to console), I decided to remove it to make testing a little easier. I could see adding it back as our needs evolve

**Comment by maxbeatty** [comment]
:shipit:

File: main.js (hunk 2 above)
Comment (robmadden) [inline main.js]: Yeah, I have no problem with that.

**Comment by robmadden** [comment]
This looks awesome. One thing, when I run the tests locally in the `refactor` branch, the tests pass, but I get the following exception:

```
console.error lib/index.js:20
      Error: testing
          at Object.mockStartRTM.mockImplementationOnce.cb (/Users/robmadden/spaceback/data/lib/index.test.js:24:50)
          at Object.mockConstructor [as startRTM] (/Users/robmadden/spaceback/data/node_modules/jest-mock/build/index.js:268:35)
          at Object.start (/Users/robmadden/spaceback/data/lib/index.js:18:3)
          at Object.test (/Users/robmadden/spaceback/data/lib/index.test.js:26:9)
```

I figured I'd ask if you were aware of this before I start digging in.

**Comment by maxbeatty** [comment]
> Error: testing

"testing" is the giveaway there. It's coming purposely from [`lib/index.test.js`](https://github.com/spbk/data/pull/3/files#diff-0ce605c4313ee8f5f153956fad34643dR24). 

This is one area where we might get back to `winston`. Jest could auto mock winston to prevent log output (couldn't easily find how to mock globals like `console` with Jest). For now, if we understand what's being outputted and why it's "good enough"

**Comment by robmadden** [comment]
Ah, right on. When I actually look at the code I see what's happening now. 😉 

Um, I don't really have a problem with this right now but I think we should probably find a better long term solution for not spewing out errors to the console when tests are run. If that's mocking or another simpler solution, fine, I don't care as long as we address it going forward.

**Pull request merged**